.azure/app-cloud-e2e.yml

```diff
@@ -112,6 +103,16 @@ jobs:
         python -m playwright install  # --with-deps
       displayName: 'Install Playwright system dependencies'
 
+    # The magic happens here it doesn't need to install the quick start dependencies.
+    # This test is very important to test the main user story of lightning app.
+    # It also e2e tests running on cloud without installing dependencies.
+    - bash: |
+        rm -rf examples/lightning-quick-start || true
+        git clone https://github.com/Lightning-AI/lightning-quick-start examples/lightning-quick-start
```

Comment:
not very happy about loading external repo with example, it shall live here so any refactoring are applied

Reply:
do you mean moving the `quick-start` example to the lightning repo ? 
@tchaton what do you think ?

Reply:
yes, exactly